cmd/substreams-sink-postgres/bulk_insert.go

```diff
@@ -0,0 +1,142 @@
+package main
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"time"
+
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+	"github.com/streamingfast/cli"
+	. "github.com/streamingfast/cli"
+	"github.com/streamingfast/cli/sflags"
+	"github.com/streamingfast/shutter"
+	sink "github.com/streamingfast/substreams-sink"
+	"github.com/streamingfast/substreams-sink-postgres/db"
+	"github.com/streamingfast/substreams-sink-postgres/sinker"
+	"go.uber.org/zap"
+)
+
+var bulkLoadCmd = Command(bulkLoadE,
+	"bulk-load <psql_dsn> <endpoint> <manifest> <module> [<start>:<stop>]",
+	"Runs first load sink process",
+	ExactArgs(6),
+	Flags(func(flags *pflag.FlagSet) {
+		sink.AddFlagsToSet(flags)
+
+		flags.Int("flush-interval", 1000, "When in catch up mode, flush every N blocks")
+		flags.Uint64("bundle-size", 1000, "Size of output bundle, in blocks")
+		flags.String("start-block", "", "Start processing at this block instead of the substreams initial block")
+		flags.String("working-dir", "./workdir", "Path to local folder used as working directory")
+		flags.String("on-module-hash-mistmatch", "error", FlagDescription(`
+			What to do when the module hash in the manifest does not match the one in the database, can be 'error', 'warn' or 'ignore'
+
+			- If 'error' is used (default), it will exit with an error explaining the problem and how to fix it.
+			- If 'warn' is used, it does the same as 'ignore' but it will log a warning message when it happens.
+			- If 'ignore' is set, we pick the cursor at the highest block number and use it as the starting point. Subsequent
+			updates to the cursor will overwrite the module hash in the database.
+		`),
+		)
+	}),
+	OnCommandErrorLogAndExit(zlog),
```

Comment:
Note to myself again, this should be needed only on the root command, investigate if there is a problem without it on each command.